state/initialize.go

```diff
@@ -123,6 +123,8 @@ func convertCloudCredentialToState(tag names.CloudCredentialTag, cloudCredential
 	credential.Owner = tag.Owner().Id()
 	credential.Cloud = tag.Cloud().Id()
 	credential.DocID = cloudCredentialDocID(tag)
+	credential.Invalid = cloudCredential.Invalid
```

Comment:
If these values are ignored, why set them here?

Reply:
This is a conversion function, from one type to another. This allows to actually pass a value for setting 'invalid' flag.